packages/console/src/pages/Users/index.tsx

```diff
@@ -41,13 +40,11 @@ const Users = () => {
           overlayClassName={modalStyles.overlay}
         >
           <CreateForm
-            onClose={(createdUser) => {
+            onClose={(createdUser, password) => {
               setIsCreateFormOpen(false);
 
-              if (createdUser) {
-                void mutate(conditional(data && [...data, createdUser]));
-
-                navigate(`/applications/${createdUser.id}/get-started`);
+              if (createdUser && password) {
+                navigate(`/users/${createdUser.id}?password=${password}`);
```

Comment:
should we close the Modal even with empty createdUse & password?

Reply:
Yes, use may click "close" button and trigger "onClose" here, so `setIsCreateFormOpen(false)` will be excecuted anyway.